packages/vite-plugin-cloudflare/src/miniflare-options.ts

```diff
@@ -506,36 +506,11 @@ export async function getDevMiniflareOptions(config: {
 										__VITE_INVOKE_MODULE__: async (request) => {
 											const payload =
 												(await request.json()) as vite.CustomPayload;
-											const invokePayloadData = payload.data as {
-												id: string;
-												name: string;
-												data: [string, string, FetchFunctionOptions];
```

Comment:
The `FetchFunctionOptions` type import needs to be removed from this file now in order for the checks to pass.